pkg/operatorsource/configuring_test.go

```diff
@@ -85,19 +89,19 @@ func TestReconcile_AlreadyConfigured_NoActionTaken(t *testing.T) {
 		Name:    phase.Succeeded,
 		Message: phase.GetMessage(phase.Succeeded),
 	}
+	scheme := runtime.NewScheme()
+	marketplace.AddToScheme(scheme)
 
 	datastore := mocks.NewDatastoreWriter(controller)
-	kubeclient := mocks.NewKubeClient(controller)
-
-	reconciler := operatorsource.NewConfiguringReconciler(helperGetContextLogger(), datastore, kubeclient)
 
-	ctx := context.TODO()
+	// We expect that the given CatalogConfigSource object already exists.
 	opsrcIn := helperNewOperatorSourceWithPhase("marketplace", "foo", phase.Configuring)
```

Comment:
If we expect the given `CatalogConfigSource` object to already exist then we should call `fakeClient.Create` before the function being tested is called, right?

Reply:
You can also initialize the fake client with runtime objects, which I have done on line 100.